### ISSUE TITLE
format negative duration without trim

### ISSUE THREAD

**Issue by firsttris** [jsmreese/moment-duration-format#104]
holla,

nice plugin! here is my issue / question:

How to format negative duration without trim?
i want to get -00:06 instead i get 00:-06

```
const test = moment.duration(-60*60*100).format('hh:mm', {
    trim: false
});
console.log(test);
>>> 00:-06
```

kind regards
Tristan

**Comment by jsmreese** [comment]
Hmmm. I'd have sworn I was testing cases like that, but apparently not.
I'm not sure what's going on, but hopefully I can take a look at it and publish an updated version this evening.

**Comment by jsmreese** [comment]
@firsttris I've published version 2.2.2 with this fix.
Please let me know if you see any other issues around negative durations.

**Comment by firsttris** [comment]
thx @jsmreese

v 2.2.2 resolved the issue for me.
